dist/stat_data.py

```diff
@@ -318,6 +318,9 @@ def __init__(self, name, desc, flags=''):
     CacheStat('cache_eviction_queue_not_empty', 'eviction server candidate queue not empty when topping up'),
     CacheStat('cache_eviction_server_evicting', 'eviction server evicting pages'),
     CacheStat('cache_eviction_server_slept', 'eviction server slept, because we did not make progress with eviction'),
+    CacheStat('cache_eviction_skip_pages_retry', 'skip pages to avoid retry that have been tried to evict before'),
```

Comment:
```suggestion
    CacheStat('cache_eviction_skip_pages_retry', 'skip pages that previously failed eviction and likely will again'),
```

Reply:
Fixed in f6176b656d9a0112686d210cdf8d28783dc815e8